hibernate-reactive-core/src/main/java/org/hibernate/reactive/engine/jdbc/mutation/internal/ReactiveMutationExecutorStandard.java

```diff
@@ -73,10 +77,64 @@ private ReactiveConnection connection(SharedSessionContractImplementor session)
 	@Override
 	public CompletionStage<Void> performReactiveBatchedOperations(
 			ValuesAnalysis valuesAnalysis,
-			TableInclusionChecker inclusionChecker, OperationResultChecker resultChecker,
+			TableInclusionChecker inclusionChecker,
+			OperationResultChecker resultChecker,
 			SharedSessionContractImplementor session) {
-		return ReactiveMutationExecutor.super
-				.performReactiveBatchedOperations( valuesAnalysis, inclusionChecker, resultChecker, session);
+		final PreparedStatementGroup batchedMutationOperationGroup = getBatchedPreparedStatementGroup();
+		if ( batchedMutationOperationGroup != null ) {
+			final List<PreparedStatementDetails> preparedStatementDetailsList = new ArrayList<>(
+					batchedMutationOperationGroup.getNumberOfStatements() );
+			batchedMutationOperationGroup.forEachStatement( (tableName, statementDetails) -> preparedStatementDetailsList
+					.add( statementDetails ) );
+			loop( preparedStatementDetailsList, statementDetails -> {
```

Comment:
This `loop` returns a CompletionStage that you are ignoring

Reply:
added a return